tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -85,7 +85,6 @@ stages:
         variables:
           releaseBuildVar: $[variables.releaseBuild]
           toolAbsolutePath: $(Build.SourcesDirectory)/tools/telemetry-generator
-          artifactPipeline: Build - client packages
```

Comment:
This pipeline only downloads artifacts from `Build - client packages`, referenced as its only `resource.pipeline`, so no need to put this in a variable (like other pipelines do because they might download from `Build - client packages` or `Build - azure`).